agent/reconciler/host_reconciler.go

```diff
@@ -285,6 +285,7 @@ func (r *HostReconciler) hostCleanUp(ctx context.Context, byoHost *infrastructur
 				r.Recorder.Event(byoHost, corev1.EventTypeWarning, "UninstallScriptExecutionFailed", "uninstall script execution failed")
 				return err
 			}
+			conditions.MarkFalse(byoHost, infrastructurev1beta1.K8sComponentsInstallationSucceeded, infrastructurev1beta1.K8sNodeAbsentReason, clusterv1.ConditionSeverityInfo, "")
 		} else {
 			err = r.uninstallk8sComponents(ctx, byoHost)
```

Comment:
should we mark False after this call too? Because currently we are having 2 ways to uninstall

Reply:
We are going to deprecate this code in #580 very soon so I thought less code to delete 🙂

Reply:
I we add that we need another test to to maintain code coverage and then delete it again.